manifests/params.pp

```diff
@@ -6,7 +6,11 @@
   $graphite_host = '127.0.0.1'
   $graphite_port = 2003
 
-  if $::is_pe {
+  $graphite_prefix = '' # lint:ignore:empty_string_assignment
+  $graphite_suffix = 'puppet'
+  $graphite_reverse_hostname = true
+
+  if str2bool($::is_pe) { # Facts are ALWAYS strings !!
     $config_file  = '/etc/puppetlabs/puppet/graphite.yaml'
```

Comment:
Facts are not always strings, `if ::is_pe` will test properly as a boolean in puppet 3x and in puppet 4x (and later versions of facter), typing of facts is possible.  This looks like a regression.